aws-predictions/src/main/java/com/amplifyframework/predictions/aws/configuration/IdentifyLabelsConfiguration.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.aws.configuration;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.amplifyframework.predictions.aws.NetworkPolicy;
+import com.amplifyframework.predictions.models.LabelType;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Configures the behavior for identifying labels.
+ */
+public final class IdentifyLabelsConfiguration {
+    private static final String CONFIG_NAME = "identifyLabels";
+    private final LabelType type;
+    private final NetworkPolicy networkPolicy;
+
+    private IdentifyLabelsConfiguration(
+            LabelType type,
+            NetworkPolicy networkPolicy
+    ) {
+        this.type = type;
+        this.networkPolicy = networkPolicy;
+    }
+
+    /**
+     * Construct an instance of {@link IdentifyLabelsConfiguration} from
+     * plugin configuration JSON object.
+     * @param configurationJson the plugin configuration
+     * @return the configuration for label identification
+     * @throws JSONException if identify configuration is malformed
+     */
+    @Nullable
+    public static IdentifyLabelsConfiguration fromJson(@NonNull JSONObject configurationJson) throws JSONException {
+        if (!configurationJson.has(CONFIG_NAME)) {
+            return null;
+        }
+
+        JSONObject identifyLabelsJson = configurationJson.getJSONObject(CONFIG_NAME);
+        String typeString = identifyLabelsJson.getString("type");
```

Comment:
This is weird, having one of these in a const, and the other not. I'd stick with a single approach, whichever you choose.

Reply:
I simply wanted to make `CONFIG_NAME` a constant because it gets used twice in the code instead of once like the other keys haha. Also, they denote different levels in the JSON document, so I figured the lowest level nodes don't deserve a special treatment.